docs/terraform-provider-edgedelta.md

```diff
@@ -188,6 +188,38 @@ The client has a number of functions, the detailed function information can be f
 |CreateMonitor|`monitors`|**monitor**: [Monitor](../edgedelta/types.go)|[*CreateMonitorResponse](../edgedelta/types.go)|
 |UpdateMonitorWithID|`monitors`|**monitorID**: `string` <br><br>  **monitor**: [Monitor](../edgedelta/types.go)|[*UpdateMonitorResponse](../edgedelta/types.go)|
 
+## Running the Provider Locally
```

Comment:
thanks for documenting this. I want to try it when I get a chance. will let you know how it goes